app/controllers/idv/in_person/ssn_controller.rb

```diff
@@ -101,11 +101,7 @@ def analytics_arguments
 
       def confirm_in_person_address_step_complete
         return if flow_session[:pii_from_user] && flow_session[:pii_from_user][:address1].present?
-        if IdentityConfig.store.in_person_residential_address_controller_enabled
-          redirect_to idv_in_person_proofing_address_url
-        else
-          redirect_to idv_in_person_step_url(step: :address)
```

Comment:
I confirmed- all instances of idv_in_person_step_url for address have been cleaned up.